proxy/proxy_test.go

```diff
@@ -2009,29 +2009,30 @@ func TestAccessLogOnFailedRequest(t *testing.T) {
 
 	rsp, err := http.Get(ps.URL)
 	if err != nil {
-		t.Error(err)
+		t.Errorf("Failed to GET: %v", err)
 		return
 	}
 
 	defer rsp.Body.Close()
 
 	if rsp.StatusCode != http.StatusBadGateway {
-		t.Error("failed to return 502 Bad Gateway on failing backend connection")
+		t.Errorf("failed to return 502 Bad Gateway on failing backend connection: %d", rsp.StatusCode)
 	}
 
+	time.Sleep(time.Millisecond)
 	output := buf.String()
 
 	proxyURL, err := url.Parse(ps.URL)
 	if err != nil {
-		t.Error(err)
+		t.Errorf("Failed to parse url: %v", err)
 		return
 	}
 
 	expected := fmt.Sprintf(`"GET / HTTP/1.1" %d %d "-" "Go-http-client/1.1"`, http.StatusBadGateway, len(http.StatusText(http.StatusBadGateway))+1)
 	if !strings.Contains(output, expected) || !strings.Contains(output, proxyURL.Host) {
-		t.Error("failed to log access", output, expected)
-		t.Log(output)
-		t.Log(expected)
+		t.Errorf("Failed to get accesslog '%v' '%v'", output, expected)
+		t.Logf("out: '%v'", output)
+		t.Logf("expect: '%v'", expected)
```

Comment:
This is redundant, values would be logged by the above Errof call

Reply:
yes, I changed it to have a cmp.Diff to spot diffs maybe easier